x/evm/statedb/statedb.go

```diff
@@ -118,7 +118,6 @@ func (s *StateDB) AddLog(log *ethtypes.Log) {
 	s.journal.append(addLogChange{})
 
 	log.TxHash = s.txConfig.TxHash
-	log.BlockHash = s.txConfig.BlockHash
```

Comment:
I didn't know we can fill in api, we can save db size but will it be slower right?

Reply:
most of the case, we don't add extra rpc queries, except get logs by block range

Reply:
seems eth filters also need

Reply:
yeah, still looking into that, the websocket part

Reply:
the other thing is, we seems duplicated the eth logs in tx response and events, both are contained in the tx result.

Reply:
not sure if we could just fill in [FilterLogs](https://github.com/crypto-org-chain/ethermint/blob/31a2b472b714308f916418d87d52a73c1cb3a3aa/rpc/namespaces/ethereum/eth/filters/utils.go#L31) for 1st part